### PR TITLE
tailwindcss setup done

### PR DESCRIPTION
**Pull request by marcinel123**
Hi Adam,

I uninstalled and installed again tailwind and postcss. After that I had to close and re-run npm start and it worked.

Regards,
Martin

tailwind_new

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -37,13 +37,15 @@
   },
   "devDependencies": {
     "@typescript-eslint/eslint-plugin": "^5.54.0",
+    "autoprefixer": "^10.4.13",
```
Comment (AdamKniec) [inline package.json:40]: why is this package necessary ? 🤔

File: src/App.tsx [hunk 2]
```diff
@@ -1,3 +1,7 @@
+import React from "react"
+import { Hello } from "./Hello";
 export const App = () => {
-  return <div className="text-lg">Ip-address-tracker!!</div>;
+  return <div className="text-6xl">Ip-address-tracker!!
```
Comment (AdamKniec) [inline src/App.tsx:4]: this formatting look really bad. Try to configure linter to wrap jsx like

return (
    your jsx code
  )

File: src/Hello.tsx [hunk 3]
```diff
@@ -0,0 +1,6 @@
+import React from "react"
```
Comment (AdamKniec) [inline src/Hello.tsx:1]: add empty line between import and component. Linter will help you with that

File: package.json (hunk 1 above)
Comment (marcinel123) [reply package.json:40]: fixed

File: src/App.tsx (hunk 2 above)
Comment (marcinel123) [reply src/App.tsx:4]: It should work fine now.

File: src/Hello.tsx (hunk 3 above)
Comment (marcinel123) [reply src/Hello.tsx:1]: fixed

**Pull request merged**